modules/ROOT/pages/xml-sdk.adoc

```diff
@@ -1211,6 +1211,78 @@ To use the operations from the <<example2, example>> above in a Mule app, it is
 </mule>
 ----
 
+[[xml-sdk-tls]]
+== TLS Support
+
+Starting with Mule 4.6, XML SDK supports Transport Layer Security (TLS). Use TLS to enhance security for your app. XML SDK connectors can support the `<tls:context>` parameter in their configuration parameters.
+
+=== Parameter Declaration
+
+Use the `xmlns:tlsEnabled=true` annotation to indicate the component to support TLS. The annotated element does not necessarily have to be a top level element. In the following example, the annotated element is `<http:request-connection>` which is a child of the `<http:request-config>` top level element.
```

Comment:
```suggestion
Use the `xmlns:tlsEnabled=true` annotation to indicate the component to support TLS. The annotated element does not necessarily have to be a top-level element. In the following example, the annotated element is `<http:request-connection>` which is a child of the `<http:request-config>` top-level element:
```

Reply:
Does the annotation turn on TLS support or is it just a flag that says it updates TLS support?